cmd/nvidia-dra-controller/imex.go

```diff
@@ -249,10 +266,10 @@ func (m *ImexManager) streamImexDomains(ctx context.Context) (<-chan string, <-c
 }
 
 // generateImexChannelPool generates the contents of a ResourceSlice pool for a given IMEX domain.
-func generateImexChannelPool(imexDomain string, numChannels int) resourceslice.Pool {
-	// Generate dchannels from 0 to numChannels
+func generateImexChannelPool(imexDomain string, cliqueid int) resourceslice.Pool {
+	// Generate dchannels from ImexChannelLimit*(cliqueid-1) to ImexChannelLimit*(cliqueid)
```

Comment:
I think this function signature should remain mostly unchanges with the addition of a "start" variable in addition to the `numChannels`.